ui/apps/platform/src/Containers/Clusters/cluster.helpers.tsx

```diff
@@ -187,13 +187,17 @@ const upgradeStates: UpgradeStates = {
         type: 'current',
     },
     MANUAL_UPGRADE_REQUIRED: {
-        displayValue: 'Manual upgrade required',
+        displayValue: 'Version not managed by ACS',
```

Comment:
Piotr, how does **not managed by ACS** here and in corresponding backend messages compare to **managed by external tools**?

The reason I ask is that even if we replace **ACS** with **RHACS** for consistency with elsewhere in UI, we avoid proper name unless necessary for meaning, because of product branding for downstream versus upstream.

Reply:
I updated the PR description with the screenshots - please have a look.